packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -51,7 +47,15 @@ exports[`Theme matches the default theme 1`] = `
     "bgError": "#FEF2F2",
     "bgInfo": "#EBEDFF",
     "bgPrimary": "#FFFFFF",
+    "bgPrimaryActive": "#F8FAFC",
```

Comment:
I noticed that besides these ones, we also have a few tokens related to `bg-branded` [here](https://www.figma.com/design/uDEjU77uIrQ5T1csn8IvIJ/Pluto-Tokens?node-id=2755-457&m=dev). Are you going to add them to this PR as well?

Reply:
I prefer doing that in a different one because all the cache crazyness on github actions